test/index.js

```diff
@@ -1,21 +1,20 @@
+/* global describe, it */
+
 var assert = require('assert')
-var Bip38 = require('../')
+var bip38 = require('../')
+var bs58check = require('bs58check')
 var fixtures = require('./fixtures')
-
-/* global beforeEach, describe, it */
+var wif = require('wif')
 
 describe('bip38', function () {
   this.timeout(70000)
 
-  var bip38
-  beforeEach(function () {
-    bip38 = new Bip38()
-  })
-
   describe('decrypt', function () {
     fixtures.valid.forEach(function (f) {
       it('should decrypt ' + f.description, function () {
-        assert.equal(bip38.decrypt(f.bip38, f.passphrase), f.wif)
+        var result = bip38.decrypt(f.bip38, f.passphrase)
+
+        assert.equal(wif.encode(0x80, result.d, result.compressed), f.wif)
```

Comment:
This is about the only real pain point IMHO,  but,  it should be simple for most users to navigate.
Thoughts @jprichardson ?